### PR TITLE
Don't register tests as filters

### LINKED ISSUE #55319: template contains deprecated call to be removed in 2.9
##### SUMMARY
template contains call to Display.deprecated or AnsibleModule.deprecate and is scheduled for removal

```
lib/ansible/template/__init__.py:192:8: ansible-deprecated-version Deprecated version ('2.9') found in call to Display.deprecated or AnsibleModule.deprecate
```

##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
```
lib/ansible/template/__init__.py
```

##### ANSIBLE VERSION
```
2.9
```

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
N/A

##### STEPS TO REPRODUCE
N/A

##### EXPECTED RESULTS
N/A

##### ACTUAL RESULTS
N/A

### PR DESCRIPTION
**Pull request by sivel**
##### SUMMARY
Don't register tests as filters. Fixes #55319

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
```
lib/ansible/template/__init__.py
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by jamescassell — APPROVED** [review]
shipit

File: docs/docsite/rst/dev_guide/testing/sanity/no-tests-as-filters.rst [hunk 1]
```diff
@@ -1,10 +0,0 @@
-Sanity Tests » no-tests-as-filters
```
Comment (mattclay) [inline docs/docsite/rst/dev_guide/testing/sanity/no-tests-as-filters.rst]: I'd keep this file around even though the test has been deleted. That way links from old test runs will continue to work. I've done this for several other tests which no longer exist.

Comment (sivel) [reply docs/docsite/rst/dev_guide/testing/sanity/no-tests-as-filters.rst]: Ok, I can do that.  I'll just have to indicate it is an orphan

**Review by mattclay — APPROVED** [review]

**Comment by acozine** [comment]
+1 from the docs team

**Pull request merged**